cmake/modules/OmrPlatform.cmake

```diff
@@ -112,6 +112,12 @@ macro(omr_platform_global_setup)
 	if(COMMAND omr_toolconfig_global_setup)
 		omr_toolconfig_global_setup()
 	endif()
+
+	# Create default implementation of omr_process_exports
+	if(NOT COMMAND omr_process_exports)
+		function(omr_process_exports)
```

Comment:
Is omr_process_exports a toolchain only function?

Reply:
@dnakamura ^^^^ ?

Reply:
I'm not entirely sure I understand what you mean. My logic is that it would be implemented at the toolchain level (since each toolchain would require different logic for exporting symbols). However I also want to create a default implementation so that a toolchain file does not need to implement the function if nothing is required to export symbols (ie linux)

Reply:
I am not sure I like the empty declaration here. I guess it beats the alternative of having to check to see if the function is defined at all of the callers.  If there was only going to be one caller I would suggest we just move the check there but I believe this is required by every share library correct?

Reply:
Correct. But i was thinking on it, and I believe I'm going to do some refactoring. There will be a `omr_process_exports` function which performs some basic housekeeping (ensuring arguments are sane, etc) which would then call the toolchain specific function.